cli/src/funTest/kotlin/OrtMainFunTest.kt

```diff
@@ -65,6 +67,10 @@ class OrtMainFunTest : StringSpec() {
         outputDir = createTestTempDir()
     }
 
+    override suspend fun afterTest(testCase: TestCase, result: TestResult) {
+        outputDir.safeDeleteRecursively(force = true)
+    }
+
```

Comment:
This is not needed, `createTestTempDir` does this automatically.

Reply:
I have removed that commit.